lib/appsignal/heartbeat.rb

```diff
@@ -0,0 +1,71 @@
+# frozen_string_literal: true
+
+require "net/http"
+require "json"
+
```

Comment:
These two requires aren't used here and are already included in the transmitter.rb file this class relies on to send requests.

```suggestion
```

Reply:
Thank you for noticing that! (I started by modifying @matsimitsu's implementation to use `Net::HTTP` instead of RestClient, and then I switched to using the processor, but I forgot to remove these imports)